me.gladwell.eclipse.m2e.android/plugin.xml

```diff
@@ -1,51 +1,54 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
This can be modified to simulate Unix end-of-line delimiters:

http://stackoverflow.com/a/1892676/48611

Not sure how we can share this setting as we do with the formatter.xml.